### PR TITLE
FLUID-6456: Remove support for obsolete ${} style interpolation in general component options

### PR DESCRIPTION
**Pull request by amb26**

### CODE REVIEW

**Comment by amb26** [comment]
ok to test

**Comment by the-t-in-rtf** [comment]
npm error.  OK to test.

**Comment by amb26** [comment]
ok to test

**Comment by amb26** [comment]
ok to test

**Comment by the-t-in-rtf** [comment]
https://github.com/fluid-project/infusion/pull/978/files#diff-ab93103a69113cb998d7630a3c4f4451R89

Shouldn't we also test the resolver syntax, i.e. ${{nonexistent}.thing}?

File: tests/framework-tests/core/js/FluidIoCTests.js [hunk 1]
```diff
@@ -77,6 +77,19 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
...
+    /** FLUID-6456 Obsolete ${} EL style tests */
+
+    fluid.defaults("fluid.tests.FLUID6456", {
+        gradeNames: "fluid.component",
+        nonexistent: "${nonexistent.thing}"
+    });
+
+    jqUnit.test("Obsolete ELstyle test", function () {
+        var that = fluid.tests.FLUID6456();
+        jqUnit.assertEquals("Should have received component with unexpanded obsolete EL", "${nonexistent.thing}",
```
Comment (the-t-in-rtf) [inline tests/framework-tests/core/js/FluidIoCTests.js]: Shouldn't we also test the resolver syntax, i.e. ${{nonexistent}.thing}?

**Comment by jobara** [comment]
Merged at c3c22dc4c3a3683280989534bb5f03f8d90d1bf9

**Pull request merged**